autoload/neomake.vim

```diff
@@ -658,10 +665,12 @@ function! s:AddExprCallback(jobinfo, prev_index) abort
     let ignored_signs = 0
     let s:postprocess = get(maker, 'postprocess', function('neomake#utils#CompressWhitespace'))
     let debug = get(g:, 'neomake_verbose', 0) >= 3
+    let custom_qf = get(g:, '_neomake_qf_enabled', 0)
```

Comment:
Sounds good to me.  Given the scope of this plugin, it could also be defined as `neomake#TitleCase` functions are documented, while `neomake#snake_case` is intended to be undocumented/subject to change.